Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/Indexer.kt

```diff
@@ -737,12 +743,8 @@ internal class NativeIndexImpl(val library: NativeLibrary) : NativeIndex() {
                 }
             }
 
-            CXIdxEntity_ObjCClass -> {
-                if (isAvailable(cursor) &&
-                        cursor.kind != CXCursorKind.CXCursor_ObjCClassRef /* not a forward declaration */) {
-
-                    getObjCClassAt(cursor)
-                }
+            CXIdxEntity_ObjCClass -> if (cursor.kind != CXCursorKind.CXCursor_ObjCClassRef /* not a forward declaration */) {
```

Comment:
Maybe `isForwardDeclaration(cursor)`?

Reply:
This is not correct, because cursor can have `CXCursor_ObjCClassRef` kind in different cases.